src/ripple/app/main/Application.cpp

```diff
@@ -1105,7 +1105,12 @@ bool ApplicationImp::setup()
     m_overlay->setupValidatorKeyManifests (*config_, getWalletDB ());
 
     {
-        auto setup = setup_ServerHandler(*config_, std::cerr);
+        std::stringstream parseErr;
+        auto setup = setup_ServerHandler(*config_, parseErr);
+        if(parseErr.rdbuf()->in_avail())
+        {
+            JLOG(m_journal.error()) << parseErr.str();
```

Comment:
Here's an example of a `std::ostream` compatible implementation that you can crib ideas from:
https://github.com/vinniefalco/Beast/blob/master/extras/beast/unit_test/dstream.hpp#L86